src/SFA.DAS.EmployerCommitmentsV2.Web/Models/Apprentice/ApprenticeshipDetailsCsvModel.cs

```diff
@@ -76,5 +79,16 @@ private static string GenerateAlerts(IEnumerable<Alerts> alerts)
 
             return alertString;
         }
+
+        private static string ToDeliveryModelDescription(DeliveryModel deliveryModel)
+        {
+            return deliveryModel switch
+            {
+                CommitmentsV2.Types.DeliveryModel.FlexiJobAgency => "Flexi-job agency",
```

Comment:
It would be better to use the existing conversion rather than duplicate it

https://github.com/SkillsFundingAgency/das-employercommitments-v2/blob/752861079271d030e7835f658c257b9f6ea100c7/src/SFA.DAS.EmployerCommitmentsV2.Web/Extensions/DeliveryModelExtension.cs#L10-L16

Reply:
yes good point.